integration/tests/user_scenario/basic_workflow/single_env_purge_unmanaged_modules.rb

```diff
@@ -76,7 +76,7 @@ class { 'motd':
 end
 
 step 'Use r10k to Purge Unmanaged Modules'
-on(master, "#{r10k_fqp} puppetfile purge -v --puppetfile #{puppet_file_path} --moduledir #{moduledir}")
+on(master, "#{r10k_fqp} puppetfile purge --puppetfile #{remote_puppetfile_path} --moduledir #{moduledir} --verbose")
```

Comment:
Seems like this change exposes the fact that we're using the Puppetfile in the "remote" to purge content in the "local" environment. Should we move to using the "local" Puppetfile in that environment?

Reply:
yep, this was part of the fun debugging I was doing last night, but I was going to make functional changes in a separate PR.

I haven't quite gotten to what the actual Puppetfile should be. I found this documentation that contradicts some of the work you did recently, I think: https://github.com/puppetlabs/r10k/blob/main/doc/puppetfile.mkd#commands

Reply:
what contradicts recent work?

Reply:
> Puppetfile subcommands assume that the Puppetfile to operate on is in the current working directory and modules should be installed in the 'modules' directory relative to the current working directory.

Reply:
how's that contradict recent work?

Reply:
idk this implied to me that the Puppetfile could be anywhere: https://github.com/puppetlabs/r10k/commit/46fb24e6a00ab81ae6c8dc0136dd6d92d7fa37a6

but not really planning to figure that out right now, if I don't have to.

Reply:
okay, turns out I read things wrong / didn't read some things at all 😅 and what Puppetfile paths we allow for the `purge` command hasn't changed, so ignore all that 🙃